_maps_2024/27-jack.md

```diff
@@ -0,0 +1,9 @@
+---
+day: 27
+creator: Ji Jin (Jack) Chen
+title: Memorial West Park, Vancouver
+thumbnail: /assets/thumbnails/2024/27-Jack-Micromapping.jpg
+source: https://github.com/jijinjc/30DayMappingChallenge2024/blob/main/Micromapping-27.Rmd
+---
+
+This is Memorial West Park in Vancouver, a park that I have lived next to since 2014, and this map was created to get revenge at a mapping project that I failed in grade 5. To recreate the amenities in the park as accurate as possible, I referred to the geojson maps and created these polygons by hand. I do feel that the coloring of the map is quite bland, and so to spice it up and create a dichotomy, I wanted to use an extremely fancy text. Of course, there is a lot on the screen, but it highlights how many activities there are at this very small park.
```

Comment:
```suggestion
This is Memorial West Park in Vancouver, a park that I have lived next to since 2014, and this map was created to get revenge at a mapping project that I failed in grade 5. To recreate the amenities in the park as accurate as possible, I referred to the geojson maps and created these polygons by hand. I do feel that the coloring of the map is quite bland, and so to spice it up and create a dichotomy, I wanted to use an extremely fancy text. Of course, there is a lot on the screen, but it highlights how many activities there are at this very small park.

![Map of Memorial West Park, Vancouver](assets/thumbnails/2024/27-Jack-Micromapping.jpg)
```